src/__internal__/focus-trap/focus-trap.spec.tsx

```diff
@@ -86,37 +85,35 @@ const MockComponent = ({
   );
 };
 
+const defaultChildren = (
+  <>
+    <button type="button">{BUTTON_ONE}</button>
+    <button type="button">{BUTTON_TWO}</button>
+  </>
+);
+
+const renderMockComponent = (props: Partial<MockComponentProps> = {}) => {
+  const children = props.children || defaultChildren;
+  return render(<MockComponent {...props}>{children}</MockComponent>);
```

Comment:
While I assume it works, this whole setup reads confusingly to me. `children` is of course a prop too, so `{...props}` will pass down the children - but then were also implicitly passing `{children}` as the children. Certainly makes me scratch my head, even though the 2 different children props should always be the same if both present.

Better in my opinion would be to write it like this:

```
const renderMockComponent = ({ children = defaultChildren, ...rest }: Partial<MockComponentProps> = {}) => {
  return render(<MockComponent {...rest}>{children}</MockComponent>);
 };
```

(which I just tried and seems to work OK)